src/main/java/calculationEngine/entities/CeInventory.java

```diff
@@ -0,0 +1,95 @@
+package calculationEngine.entities;
+
+import calculationEngine.environment.CeItem;
+
+public class CeInventory {
+
+    CeItem armorShoulder;
+    CeItem armorShoes;
+    CeItem armorLegs;
+    CeItem armorChest;
+    CeItem armorHead;
+    CeItem weapon;
+
```

Comment:
This is the wrong place to do this imo. Create an Enum in the CeItem Class and give a Type to the CEItem

Reply:
rename Variable to equiped...